ztp/siteconfig-generator/siteConfig/siteConfigBuilder.go

```diff
@@ -206,7 +206,7 @@ func (scbuilder *SiteConfigBuilder) getClusterCRs(clusterId int, siteConfigTemp
 		return clusterCRs, err
 	}
 
-	deprecationWarnings := getDeprecationWarnings(cluster)
+	deprecationWarnings := getDeprecationWarnings(cluster, deprecateCPUset, deprecateExtraManifestPath)
```

Comment:
deprecateCPUset and deprecateExtraManifestPath are constants no need to pass them as variable, just assign them.